pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "gatorgrade"
+version = "0.1.0"
+description = "Python tool to execute GatorGrader"
+authors = ["Michael Abraham", "Jacob Allebach", "Liam Black", "Katherine Burgess", "Yanqiao Chen", "Ochirsaikhan Davaajambal", "Tuguldurnemekh Gantulga", "Anthony Grant-Cook", "Dylan Holland", "Peyton Kelly", "Luke Lacaria", "Lauren Nevill", "Jack Turner", "Daniel Ullrich", "Garrison Vanzin", "Rian Watson"]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+typer = "^0.4.1"
+
+[tool.poetry.dev-dependencies]
+taskipy = "^1.10.1"
+pytest = "^7.1.1"
+pytest-cov = "^3.0.0"
+pylint = "^2.13.4"
+black = "^22.3.0"
+flake8 = "^4.0.1"
+mypy = "^0.942"
+pydocstyle = "^6.1.1"
+
+[tool.taskipy.tasks]
+test = { cmd = "pytest --cov-report term-missing --cov-fail-under=70  --cov-branch --cov=package tests/", help = "Run the pytest test suite" }
+lint = { cmd = "pylint package/ tests/" }
```

Comment:
Should `package/` be `gatorgrade` in light of the structure that was proposed in the #sep channel?

Reply:
Yes, I will change it to reflect that. Thank you for bringing it to light